net-p2p/amule/amule-2.3.2-r4.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+WX_GTK_VER="3.0-gtk3"
+
+inherit wxwidgets user
+
+MY_P="${PN/m/M}-${PV}"
+
+DESCRIPTION="aMule, the all-platform eMule p2p client"
+HOMEPAGE="http://www.amule.org/"
+SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.xz"
```

Comment:
Hmm, I dont think this `mirror://sourceforge:` lists mirrors anymore but points to https://download.sourceforge.net, so you could replace it and we can get rid of the mirror file at some point.